cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -585,6 +586,102 @@ class SimpleArray
         }
     }
 
+    void sort(void)
+    {
+        if (ndim() != 1)
+        {
+            throw std::runtime_error("SimpleArray: Sorting is only supported in 1D array.");
```

Comment:
Add the dimension number in the error message.  See the example: https://github.com/solvcon/modmesh/blob/74a3a0f7e91f85638303457bc755cd0fd51edc59/cpp/modmesh/buffer/SimpleArray.hpp#L306

Do not capitalize after `:`.  That is, write `SimpleArray: sorting`.